CONTRIBUTING.md

```diff
@@ -0,0 +1,33 @@
+# Contributing to Atlas
+
+:tada: To begin with, thank you so much in your interest in contributing to Atlas :tada:
+
+### Contributing TL;DR
+
+1. Always make sure to create an issue/comment on an existing issue before you start any work.
```

Comment:
```suggestion
1. Always make sure to create an issue or comment on an existing issue before you start any work.
```
A small thing, but I don't like `/` here, this could suggest that you can `create an issue on an existing issue` :D I mean I read that first time, had to go back and read again to understand the whole sentence.